### PR TITLE
feat: change wording to 'passcode'

### LINKED ISSUE #2251: Change unlock password to "unlock passcode"
Users confuse unlock password with Alby Account password. This minor copy change from "password" to "passcode" may help a bit pointing out the difference. 

I also prepared a new illustration for onboarding form that will better show how it works:

image

With that change I also think we can improve copy here a bit (I do not think mentioning Alby Account here serves purpose, as users still confuse this and many users will not use Alby Account at all or don't know yet what is it.). See screenshots (note: second version screenshot is based on [new proposed layout](https://github.com/getAlby/lightning-browser-extension/issues/2250), but changes to copy and screenshot can be applied to current layout as well)

Current: 
**Set an unlock password**
This password is used to protect your wallet and provide access to the browser extension. It cannot be reset and is separate from your Alby account password.

Screenshot 2023-03-22 at 02 05 33

Proposed: 
**Set an unlock code**
This passcode will allow you to unlock Alby on this device.  It cannot recovered, but may be changed later.

Screenshot 2023-03-22 at 02 07 29

### PR DESCRIPTION
**Pull request by im-adithya**
### Describe the changes you have made in this PR

Changes the unlock screenshot and the wording from "password" to "passcode" as an attempt to help users understand better than this is not related to their alby account's password.

### Link this PR to an issue [optional]

Fixes #2251 

### Type of change

(Remove other not matching type)

- `feat` or `chore`

### Screenshots of the changes [optional]

<p  style="display: flex; justify-content: center;">

</p>

### How has this been tested?

Manually

### Checklist

- [x] My code follows the style guidelines of this project and performed a self-review of my own code
- [x] New and existing tests pass locally with my changes
- [x] I checked if I need to make corresponding changes to the documentation (and made those changes if needed)

### CODE REVIEW

**Comment by im-adithya** [comment]
@stackingsaunter should be good now!

File: static/assets/images/unlock_passcode.png
Comment (reneaaron) [inline static/assets/images/unlock_passcode.png:1]: Please optimize PNGs using some tool like tinypng. Lossless compression can save ~80% here.

**Review by reneaaron — CHANGES_REQUESTED** [review]
See comment above, other than that :ok_hand: 

tACK

**Comment by im-adithya** [comment]
Done!

File: static/assets/images/unlock_passcode.png
Comment (im-adithya) [reply static/assets/images/unlock_passcode.png:1]: Done!

**Review by reneaaron — APPROVED** [review]

**Pull request merged**